app/controllers/posts_controller.rb

```diff
@@ -8,7 +8,30 @@ def index
   def show
     author_id = posted_by
     @user = User.find(author_id)
-    @post = Post.where(id: params[:id], author_id:).order(id: :DESC).first
+    @post = Post.where(author_id:).first
+    # @post = Post.includes(:comments).where(id: params[:id], author_id:).order('comments.id DESC').first
```

Comment:
- [ ] Please remove the following commented codes in order to make your codebase clean. This is among the best coding practices that help to code efficiently 😃 👍

Reply:
Thank you